src/saltext/vmware/states/vmc_nat_rules.py

```diff
@@ -0,0 +1,481 @@
+
+        .. code-block::
+
+            tags:
+              - tag: <tag-key-1>
+                scope: <tag-value-1>
+              - tag: <tag-key-2>
+                scope: <tag-value-2>
+
+    sequence_number
+        (Optional) Sequence number of the nat rule
+            The sequence_number decides the rule_priority of a NAT rule.
+            default: 0
+            type: int
+
+    display_name
+        Identifier to use when displaying entity in logs or GUI. This is applicable for only update scenario.
+        For create scenario, display_name would be same as rule_id.
+
+    Example Values:
+
+        .. code-block::
+
+            action: REFLEXIVE
+            translated_network: 10.182.171.36
+            translated_ports: null
+            destination_network: ''
+            source_network: 192.168.1.23
+            sequence_number: 0
+            service: ''
+            logging: false
+            enabled: false
+            scope:
+              - /infra/labels/cgw-public
+            tags:
+              - tag: tag1
+                scope: scope1
+            description: ''
+            firewall_match: MATCH_INTERNAL_ADDRESS
+    """
+
+    input_dict = {
+        "action": action,
+        "description": description,
+        "destination_network": destination_network,
+        "scope": scope,
+        "service": service,
+        "source_network": source_network,
+        "tags": tags,
+        "translated_network": translated_network,
+        "translated_ports": translated_ports,
+        "enabled": enabled,
+        "firewall_match": firewall_match,
+        "logging": logging,
+        "sequence_number": sequence_number,
+        "display_name": display_name,
+    }
+
+    input_dict = {k: v for k, v in input_dict.items() if v != vmc_constants.VMC_NONE}
+
+    get_nat_rule_response = __salt__["vmc_nat_rules.get_by_id"](
+        hostname=hostname,
+        refresh_key=refresh_key,
+        authorization_host=authorization_host,
+        org_id=org_id,
+        sddc_id=sddc_id,
+        tier1=tier1,
+        nat=nat,
+        nat_rule=nat_rule,
+        verify_ssl=verify_ssl,
+        cert=cert,
+    )
+
+    if "error" in get_nat_rule_response:
+        if "could not be found" in get_nat_rule_response["error"]:
+            get_nat_rule_response = None
+        else:
+            return vmc_state._create_state_response(
+                name=name, comment=get_nat_rule_response["error"], result=False
+            )
+
+    if __opts__.get("test"):
+        log.info("present is called with test option")
+        if get_nat_rule_response:
+            return vmc_state._create_state_response(
+                name=name, comment="State present will update nat rule {}".format(nat_rule)
+            )
+        else:
+            return vmc_state._create_state_response(
+                name=name, comment="State present will create nat rule {}".format(nat_rule)
+            )
```

Comment:
**suggestion** this can be simplified as

```suggestion
        return vmc_state._create_state_response(name=name, comment="State present will {} nat rule {}".format("update" if get_nat_rule_response else "create", nat_rule))
```

Reply:
done above changes